src/cogs/fun.py

```diff
@@ -10,50 +10,29 @@ def __init__(self, bot):
     @nextcord.slash_command()
     async def bofh(self, inter: nextcord.Interaction):
         """Convenient solutions to inconvenient tech problems"""
-        # https://stackoverflow.com/questions/14924721/how-to-choose-a-random-line-from-a-text-file#14924739
-        line_num = 0
-        selected = ""
-        with open("data/excuses.txt") as f:
-            while 1:
-                line = f.readline()
-                if not line:
-                    break
-                line_num += 1
-                if random.uniform(0, line_num) < 1:
-                    selected = line
-        await inter.send(selected.strip())
+        # Wow that was really overcomplicated.
```

Comment:
No need to add a comment about the previous code to the current code

Reply:
oh cmon :/